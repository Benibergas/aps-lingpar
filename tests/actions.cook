adicione farinha 100;
misture_ate massa_uniforme;
asse bolo;
sirva bolo;