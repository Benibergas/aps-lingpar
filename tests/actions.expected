Adicionando 100 de farinha
Misturando até condição 1
Assando bolo
Servindo bolo
